src/main/java/uk/gov/hmcts/ccd/data/user/UserRepository.java

```diff
@@ -17,4 +17,6 @@
     Set<SecurityClassification> getUserClassifications(String jurisdictionId);
 
     SecurityClassification getHighestUserClassification(String jurisdictionId);
+
+    String getUserName();
```

Comment:
This user name is in fact the user ID, could the function be renamed to `getUserID()`?

Reply:
👍